data_structures/linked_list/QueueLinkedList.c

```diff
@@ -0,0 +1,133 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+typedef struct queue
+{
+    int data;
+    struct queue *next;
+}queue;
+typedef struct Q
```

Comment:
As I said in your previous PR, again, please maintain at least one line between chunks of code/functions. It is important to write clean and well structured code.